### ISSUE TITLE
Basech a medal

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#443]
Maybe it means something like someone has one eye that doesn't work?

**Comment by Gnalu** [comment]
Yes, someone that has eyesight issues sues literally or someone that cannot
locate anything. Another way of saying ‘Ke cheuiko?’

**Comment by Lukes-Imeyuns** [comment]
Or they literally have cataracts.

John, with one eye that doesn't work, that would be BALECH.

Beiusech a medal on the other hand is one with keen eyesight, literally and
figuratively. :)
